02.AsyncAwait/AsyncAwait.Tasks/AsyncAwait.Task1.CancellationTokens/Program.cs

```diff
@@ -0,0 +1,79 @@
+﻿/*
+* Study the code of this application to calculate the sum of integers from 0 to N, and then
+* change the application code so that the following requirements are met:
+* 1. The calculation must be performed asynchronously.
+* 2. N is set by the user from the console. The user has the right to make a new boundary in the calculation process,
+* which should lead to the restart of the calculation.
+* 3. When restarting the calculation, the application should continue working without any failures.
+*/
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace AsyncAwait.Task1.CancellationTokens;
+
+internal class Program
+{
+    private static Task _calculator;
```

Comment:
[Optional]: Judging by the name of this field, I would expect an object/service, but not a Task.